.github/workflows/pipeline.yml

```diff
@@ -0,0 +1,94 @@
+name: Pipeline deployment
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches: [master]
+    types: [opened, synchronize]
+
+env:
+  ML_IMAGE_TAG: activelearning_ml:test
+  API_IMAGE_TAG: activelearning_api:test
+  ML_ENDPOINT: train_and_predict/
+  API: api/
+  WEBAPP: webapp/
+
+jobs:
+  linting:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.8' 
+          architecture: 'x64'
+          cache: 'pip'
+      - name: flake8 Lint
+        uses: py-actions/flake8@v2
+        with:
+          ignore: "E402"
+          max-line-length: "120"
+          path: .
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '14.x'
+      - name: eslint lint
+        working-directory: ${{ env.WEBAPP }}
+        run: |
+          npm install eslint
+          npm install @typescript-eslint/eslint-plugin@latest --save-dev
+          npm install @typescript-eslint/parser --save-dev
+          npm run lint
+
+
+  deployment:
+    runs-on: ubuntu-18.04
+    needs: [linting]
+    if: ${{ github.event_name == 'push' }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Build and push ML_ENDPOINT image
+        if: ${{ github.event_name == 'push' }}
+        uses: docker/build-push-action@v2
+        with:
+          context: ${{ env.ML_ENDPOINT }}
+          platforms: linux/amd64,linux/arm64
+          push: true
+          tags: ${{ secrets.DOCKERHUB_USERNAME }}/${{ env.ML_IMAGE_TAG }}
+      - name: Build and push API image
+        uses: docker/build-push-action@v2
+        with:
+          context: ${{ env.API }}
+          platforms: linux/amd64,linux/arm64
+          push: true
+          tags: ${{ secrets.DOCKERHUB_USERNAME }}/${{ env.API_IMAGE_TAG }}
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '14.x'
+      - name: Build and push web app to GH pages
+        working-directory: ${{ env.WEBAPP }}
+        run: |
+          npm rebuild node-sass
+          npm install
+          npm run build
+          npm run deploy
+      - uses: actions/checkout@v2
+      - name: Bump version and push tag
+        uses: anothrNick/github-tag-action@1.33.0
```

Comment:
@Molkree, I'm not able to change to version 1, think I'll just pin the latest version, which is 1.38.0?

Reply:
Right, looks like they don't provide a moving `1` tag as some other Actions do. Pin to the latest version instead of master, should be [`1.38.0`](https://github.com/anothrNick/github-tag-action/releases/tag/1.38.0).